src/ViewModel/Factory/SiteHeaderFactory.php

```diff
@@ -104,21 +104,21 @@ public function createSiteHeader(Model $item = null, Profile $profile = null) :
                             2 => $this->packages->getUrl('assets/patterns/img/patterns/molecules/nav-primary-menu-ic_2x.png'),
                             1 => $this->packages->getUrl('assets/patterns/img/patterns/molecules/nav-primary-menu-ic_1x.png'),
                         ],
-                        'Menu icon'
+                        ''
```

Comment:
👍, as it's decorative (and has text).

Reply:
Should do the same with the search icon

Reply:
The search icon change is coming as part of some work to improve the accessibility of the site header js-search interaction.